### PR TITLE
Small naming and tooltip tweaks

### PR DESCRIPTION
**Pull request by YeldhamDev**
Added end periods to some tooltips, and capitalized both instances of "root", namely, the root node.

### CODE REVIEW

File: scene/main/scene_tree.cpp [hunk 1]
```diff
@@ -1408,7 +1408,7 @@ SceneTree::SceneTree() {
 	//create with mainloop
 
 	root = memnew(Window);
-	root->set_name("root");
+	root->set_name("Root");
```
Comment (akien-mga) [inline scene/main/scene_tree.cpp:1411]: Won't that break compat with scripts that access `get_node("/root")`?

Comment (YeldhamDev) [reply scene/main/scene_tree.cpp:1411]: I'm quite sure everyone just uses `get_tree().root`, as it makes more sense.

Comment (Calinou) [reply scene/main/scene_tree.cpp:1411]: I've seen `get_node("/root")` in the wild quite often, so if we change this, we should make this prominent in the changelog.

Comment (YeldhamDev) [reply scene/main/scene_tree.cpp:1411]: That's weird that people would go for that route, but if you say so.
The main reason for this change is to make it match the other node names, as everything follows PascalCase.

Comment (akien-mga) [reply scene/main/scene_tree.cpp:1373]: I'd leave that out for this PR, this simple change is worth a GIP. :)
The rest is good to merge.

**Comment by YeldhamDev** [comment]
@akien-mga "Root" renamed back to "root".

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**